base/applications/mspaint/lang/ro-RO.rc

```diff
@@ -105,18 +105,18 @@ ID_POPUPMENU MENU
 BEGIN
     POPUP "Popup #0"
     BEGIN
-        MENUITEM "C&ut\tCtrl+X", IDM_EDITCUT
-        MENUITEM "Copy\tCtrl+C", IDM_EDITCOPY
-        MENUITEM "&Paste\tCtrl+V", IDM_EDITPASTE
-        MENUITEM "De&lete selection\tDel", IDM_EDITDELETESELECTION
-        MENUITEM "Select &All\tCtrl+A", IDM_EDITSELECTALL
+        MENUITEM "&Decupează\tCtrl+X", IDM_EDITCUT
+        MENUITEM "&Copiază\tCtrl+C", IDM_EDITCOPY
+        MENUITEM "&Lipește\tCtrl+V", IDM_EDITPASTE
+        MENUITEM "Șter&ge selecția\tDel", IDM_EDITDELETESELECTION
```

Comment:
The comment on "G" access key that got lost:
This is not a good fit for an access key, and the reason [was already mentioned before](https://github.com/reactos/reactos/pull/5564#discussion_r1295994921).